zig test fails

```
/home/runner/work/fiat-crypto/fiat-crypto/inversion/zig/inversion.zig:55:25: error: redundant comptime keyword in already comptime scope
```
@jedisct1

--- a/inversion/zig/inversion.zig
+++ b/inversion/zig/inversion.zig
@@ -52,7 +52,7 @@ fn fieldElement(comptime Field: type) type {
             return fe;
         }
 
-        pub const one = comptime one: {
+        pub const one = one: {
             var fe: Self = undefined;
             fiat.setOne(&fe.limbs);
             break :one fe;

